wfe/wfe.go

```diff
@@ -327,6 +332,9 @@ func (wfe *WebFrontEndImpl) Directory(ctx context.Context, logEvent *requestEven
 		"new-cert":    newCertPath,
 		"revoke-cert": revokeCertPath,
 	}
+	if features.Enabled(features.AllowKeyRollover) {
+		directoryEndpoints["key-change"] = rolloverPath
```

Comment:
Queries are currently taking absolutely forever to complete for some reason but it appears we have 0 requests from pre-0.6.0 versions of `certbot` currently in at least the last two weeks (which while entirely possible does seem a little suspect).